web/src/main/java/org/springframework/security/web/authentication/www/BasicAuthenticationFilter.java

```diff
@@ -183,7 +183,7 @@ protected void doFilterInternal(HttpServletRequest request,
 			SecurityContextHolder.clearContext();
 
 			if (debug) {
-				this.logger.debug("Authentication request for failed: " + failed);
+				this.logger.debug("Authentication request for failed: " + failed, failed);
```

Comment:
Please change the debug message to be the same as `BearerTokenAuthenticationFilter` https://github.com/spring-projects/spring-security/blob/56f524259588e225368327a4ee19eaaf2416ba99/oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/web/BearerTokenAuthenticationFilter.java#L135